src/js/select2/data/select.js

```diff
@@ -14,15 +14,21 @@ define([
 
   SelectAdapter.prototype.current = function (callback) {
     var data = [];
-    var self = this;
 
-    this.$element.find(':selected').each(function () {
-      var $option = $(this);
+    if (this.$element.length > 0) {
```

Comment:
I thought `$element` should always be present here? Do we have any tests where this is not the case?

Reply:
@kevin-brown basicly it was an equivalent of `$.find()` funcion call (it won't fail even if it actually doen't contain a single element). Also I don't see inside `SelectAdapter` any explicit check of whether `this.$element` contain any dom element or it is just an emply wrapper.

If you think this check is redundant, I can remove it.

Reply:
@kevin-brown what do you think? Should I remove this check?

Reply:
Sorry, lost track of this PR. I think the check can be safely removed, since Select2 will not initialize itself unless an element is present. So you can assume that `this.$element` references a DOM element.

This can be verified in the Select2 jQuery adapter file where we call `each()` on the element set to ensure that Select2 is initialized once per element and also is only initialized when there are elements.